### PR TITLE
feat: Helm flag to set the Orgs to use for workload auto import/delete

### PR DESCRIPTION
**Pull request by ivanstanev**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

The workload auto-import and auto-delete policy now comes as a preset. In order to use it, users should provide a list of Snyk Organization public IDs so that snyk-monitor knows which Orgs to use for importing or deleting scanned images.

### CODE REVIEW

File: snyk-monitor/templates/configmap.yaml [hunk 1]
```diff
@@ -3,9 +3,35 @@ apiVersion: v1
...
+    helm.sh/chart: {{ include "snyk-monitor.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
   excludedNamespaces: |-
     {{- range .Values.excludedNamespaces }}
     {{ . }}
     {{- end }}
 {{ end }}
+---
```
Comment (dkontorovskyy) [inline snyk-monitor/templates/configmap.yaml:17]: Do we need `{{ if .Values.policyOrgs }}`? Or we want this policy be set for every kubernete-monitor by default?

**Review by dkontorovskyy — COMMENTED** [review]
Nice change 👏 

Do we also need to update README.md with new Helm values?

**Comment by ivanstanev** [comment]
> Nice change 👏
> 
> Do we also need to update README.md with new Helm values?

There isn't anything in the README yet because we haven't gotten to the task of adding the docs yet - that's a pre-GA task 🙂

File: snyk-monitor/templates/configmap.yaml (hunk 1 above)
Comment (ivanstanev) [reply snyk-monitor/templates/configmap.yaml:17]: We want it set by default to make it easier for on-boarding

**Review by kat1906 — APPROVED** [review]

**Pull request merged**